resources/js/components/SnackspaceStripePayment.vue

```diff
@@ -0,0 +1,467 @@
+<template>
+  <div class="vue-remove">
+    <button type="button" class="btn btn-primary mb-1" @click="showModal"><i class="fas fa-plus" aria-hidden="true"></i> Add Money To Snackspace</button>
+
+    <!-- Card Modal -->
+    <div ref="cardModal" class="modal fade" id="addMoneyToSnackspaceModal" tabindex="false" role="dialog" aria-labelledby="addMoneyToSnackspaceLabel" aria-hidden="true">
+      <div class="modal-dialog modal-dialog-centered" role="document">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h5 class="modal-title" id="addMoneyToSnackspaceLabel">Add Money To Snackspace</h5>
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+              <span aria-hidden="true">&times;</span>
+            </button>
+          </div>
+          <div ref="modalBody" class="modal-body">
+            <p>To add money to your snack space account please select an amount, then enter your card details.</p>
```

Comment:
Add a line about why they can't choose an amount.

Maybe: "Due to processing fees, it is not economical for us to offer other amounts."